coreteam.py

```diff
@@ -43,16 +43,17 @@
 # List of PrestaShop project repositories
 #
 PROJECTS = {
-    'prestashop-specs': 'PrestaShop Specifications',
-    'docs': 'Changes in developer documentation',
-    'TranslationToolsBundle': 'TranslationTools Bundle',
     'QANightlyResults': 'QA nightly results',
-    'php-coding-standards': 'PHP Coding Standards',
-    'ps_facetedsearch': 'Faceted search module',
-    'gamification': 'Gamification module',
+    'TranslationToolsBundle': 'TranslationTools Bundle',
     'autoupgrade': 'Auto Upgrade module',
     'classic-rocket': 'Classic-rocket theme',
+    'docker': 'Docker',
+    'docs': 'Changes in developer documentation',
+    'gamification': 'Gamification module',
+    'php-coding-standards': 'PHP Coding Standards',
+    'prestashop-specs': 'PrestaShop Specifications',
     'prestashop-ui-kit': 'Prestashop UI Kit',
```

Comment:
We don't need to add the other prestashop modules?

Reply:
When we will see it's needed.